apps/projects/serializers.py

```diff
@@ -10,11 +13,47 @@ class ProjectSerializer(serializers.ModelSerializer):
     organisation = serializers.SerializerMethodField()
     tile_image = serializers.SerializerMethodField()
     tile_image_copyright = serializers.SerializerMethodField()
+    status = serializers.SerializerMethodField()
+    participation_active = serializers.SerializerMethodField()
+    participation_string = serializers.SerializerMethodField()
+    future_phase = serializers.SerializerMethodField()
+    active_phase = serializers.SerializerMethodField()
+    past_phase = serializers.SerializerMethodField()
+    offensive = serializers.SerializerMethodField()
+    comment_count = serializers.SerializerMethodField()
 
     class Meta:
         model = Project
         fields = ['title', 'url', 'organisation', 'tile_image',
-                  'tile_image_copyright']
+                  'tile_image_copyright',
+                  'status', 'access',
+                  'participation_active',
+                  'participation_string',
+                  'future_phase', 'active_phase',
+                  'past_phase',
+                  'offensive', 'comment_count']
+
+    @lru_cache(maxsize=1)
```

Comment:
What does this do? Cache the status somehow?

Reply:
This is an opteermising cache algorithum used in mB which I just copied over with some of the serializer stuff, it removes least recently used cached items, I would imagine we won't have long project lists for the mods but I would say it doesn't hurt to have it, what do you think?

Reply:
Sure, we can keep it. Was just wondering what it does, but sounds good!